.gitattributes

```diff
@@ -0,0 +1 @@
+* -text
```

Comment:
A prior commit had changed line ending to CRLF.  I will have to change it to auto when I recommit the changes